packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -1768,6 +1768,7 @@ export interface IBreadcrumbItem {
     isCurrentItem?: boolean;
     key: string;
     onClick?: (ev?: React.MouseEvent<HTMLElement>, item?: IBreadcrumbItem) => void;
+    role?: string;

```

Comment:
Looking at this now, I think maybe it'd be better to add this role to `IBreadcrumbProps`? I don't know if there'd ever be a scenario where items within the same breadcrumb would have different roles

Reply:
i do think it's more suitable to add it here, as what you have now